build/config/compiler/BUILD.gn

```diff
@@ -343,7 +343,7 @@ config("rtti") {
 }
 
 config("rtti_default") {
-  configs = [ ":no_rtti" ]
+  configs = [ ":rtti" ]
```

Comment:
This does not seem like the right change.  We only want RTTI for the python controller, not across the board, I would think.  This is leading to unacceptable codesize increases on the embedded platforms.

Reply:
Good point, I am not familiar how to set this on all libraries when building python controller. It need to be set across the board, because there will be compilation issues. If you know a way to set this when building the python controller, it would be super helpful.

Reply:
Ok fixed.